### ISSUE TITLE
about test result

### ISSUE THREAD

**Issue by sunyclj** [IanYeung/RealVSR#4]
excuse me，input size is 512*1024，after Video Super-resolution，the size is thr same as the input. i think the size should be bigger, x2 or x4 and so on.what is the reason?

**Comment by sunyclj** [comment]
When I entered my consecutive  50 frames , I got 50 frames of images, but the results were terrible ,the output is more ambiguous than the input.why?

**Comment by IanYeung** [comment]
> When I entered my consecutive 50 frames , I got 50 frames of images, but the results were terrible ,the output is more ambiguous than the input.why?

Hi, for the resolution problem, please refers to other several issues. May I know what is your input?

**Comment by fehimebetul** [comment]
I agree with the @sunyclj,  becuase SR is used for upsampling so, if we use LR and HR with the same size, What does SR algorithm do?
